pymongo/cursor.py

```diff
@@ -144,6 +144,11 @@ def __init__(self, collection, filter=None, projection=None, skip=0,
         if not isinstance(limit, int):
             raise TypeError("limit must be an instance of int")
         validate_boolean("no_cursor_timeout", no_cursor_timeout)
+        if no_cursor_timeout:
+            warnings.warn("use an explicit session with no_cursor_timeout=True"\
+                          "otherwise the cursor may still timeout after 30 minutes,"\
+                          "for more info see DOCS-11255",
```

Comment:
A few suggestions:
- replace DOCS-11255 with https://docs.mongodb.com/v4.4/reference/method/cursor.noCursorTimeout/#session-idle-timeout-overrides-nocursortimeout
- limit lines to 80 characters
- remove the trailing `\` since it's not needed.